flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -971,8 +971,11 @@ private static void setupNotificationChannel(
       notificationChannel.setDescription(notificationChannelDetails.description);
       notificationChannel.setGroup(notificationChannelDetails.groupId);
       if (notificationChannelDetails.playSound) {
+        Integer audioAttributesUsage = notificationChannelDetails.audioAttributesUsage != null
```

Comment:
As I mentioned coalescing earlier but wasn't done, perhaps you're not familiar with what that means. Rather than having

```
Integer audioAttributesUsage = notificationChannelDetails.audioAttributesUsage != null
                 ? notificationChannelDetails.audioAttributesUsage
                 : AudioAttributes.USAGE_NOTIFICATION;
```

this can be expressed as

```
Integer audioAttributesUsage = notificationChannelDetails.audioAttributesUsage ?? AudioAttributes.USAGE_NOTIFICATION;
```

Reply:
Apologies, I thought Java supported this but turns out it doesn't